calicoctl/commands/node/run.go

```diff
@@ -298,17 +298,30 @@ Description:
 		setNFConntrackMax()
 	}
 
+	// Make sure the calico-node is not already running before we attempt
+	// to start the node.
+	fmt.Println("Removing old calico-node container (if running).")
+	err = exec.Command("docker", "rm", "-f", "calico-node").Run()
+	if err != nil {
+		log.WithError(err).Debug("Unable to remove calico-node container")
```

Comment:
we should add `ignore this if calico-node is not running` so users don't get confused when they see that message in the logs